src/amo/pages/Home/index.js

```diff
@@ -248,9 +251,6 @@ export class HomeBase extends React.Component {
     // collection by index.
     const renderFeaturedCollection = (index) => {
       const metadata = featuredCollectionsMetadata[index];
-      if (metadata && metadata.isTheme) {
-        return null;
-      }
```

Comment:
I believe this was used when we had to disable themes for Android and this never got removed entirely because we usually don't have collections with themes only. There is no need to keep that or `isTheme` in the metadata.